src/Modules/DashboardModule/components/Dashboard/Dashboard.tsx

```diff
@@ -1,5 +1,209 @@
-import React from "react";
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { Bar, Doughnut } from "react-chartjs-2";
+import { useUser } from "../../../../Context/AuthContext";
+import {
+  baseUrl,
+  getRequestHeaders,
+  handleApiError,
+} from "../../../../utils/Utils";
+import Header from "../../../SharedModule/components/Header";
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  DoughnutController,
+  ArcElement,
+  Title,
+  Tooltip,
+  Legend,
+} from "chart.js";
+
+import "./Dashboard.Moule.css";
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  DoughnutController,
+  ArcElement,
+  Title,
+  Tooltip,
+  Legend
+);
 
 export default function Dashboard() {
-  return <>Dashboard</>;
+  const { userRole } = useUser();
+  const [todoCount, settodoCount] = useState<number>(0);
+  const [progressCount, setprogressCount] = useState<number>(0);
+  const [doneCount, setdoneCount] = useState<number>(0);
+  const [activatedEmployeeCount, setactivatedEmployeeCount] =
+    useState<number>(0);
+  const [deactivatedEmployeeCount, setdeactivatedEmployeeCount] =
+    useState<number>(0);
+
+  const getTasksCount = async () => {
+    try {
+      const response = await axios.get(`${baseUrl}/Task/count`, {
+        headers: getRequestHeaders(),
+      });
+
+      settodoCount(response.data.done);
+      setprogressCount(response.data.inProgress);
+      setdoneCount(response.data.toDo);
+    } catch (error) {
+      handleApiError(error);
+    }
+  };
+
+  const getUserCount = async () => {
+    try {
+      const response = await axios.get(`${baseUrl}/Users/count`, {
+        headers: getRequestHeaders(),
+      });
+      setactivatedEmployeeCount(response.data.activatedEmployeeCount);
+      setdeactivatedEmployeeCount(response.data.deactivatedEmployeeCount);
+    } catch (error) {
+      handleApiError(error);
+    }
+  };
+
+  useEffect(() => {
+    getTasksCount();
+    getUserCount();
+  }, []);
+
+  const tasksData = {
+    labels: ["To Do", "In Progress", "Done"],
+    datasets: [
+      {
+        label: "Tasks",
+        data: [todoCount, progressCount, doneCount],
+        backgroundColor: ["#FF6384", "#36A2EB", "#4BC0C0"],
```

Comment:
We will need one set for the dark mode and another one for the light mode.

Reply:
i handled the h1 of the label by css variable to apply light/dark mode